charts/operator-wandb/values.yaml

```diff
@@ -167,6 +169,49 @@ global:
     parameters: {}
     params: {}
     caCert: ""
+    certPath: "redis_ca.pem"
+    secret:
+      secretName: ""
+      secretKey: "REDIS_PASSWORD"
+
+  # (Optional) These (settingsCache, metadataCache, or taskQueue) almost never need to be set, do not set these unless you understand the risks.
+  # Settings cache and metadata cache allow specifying a non-default redis instance for these caches.  Defaults to global.redis if not specified.
+  settingsCache:
+    host: ""
+    port: 6379
+    password: ""
+    # external should remain false; it is for W&B internal use only.
+    external: false
+    parameters: {}
+    params: {}
+    caCert: ""
+    certPath: "redis_settings_cache_ca.pem"
+    secret:
+      secretName: ""
+      secretKey: "REDIS_PASSWORD"
+  metadataCache:
+    host: ""
+    port: 6379
+    password: ""
+    # external should remain false; it is for W&B internal use only.
+    external: false
+    parameters: {}
+    params: {}
+    caCert: ""
+    certPath: "redis_metadata_cache_ca.pem"
+    secret:
+      secretName: ""
+      secretKey: "REDIS_PASSWORD"
+  taskQueue:
+    host: ""
+    port: 6379
+    password: ""
+    # external should remain false; it is for W&B internal use only.
+    external: false
+    parameters: {}
+    params: {}
+    caCert: ""
+    certPath: "redis_task_queue_ca.pem"
     secret:
       secretName: ""
       secretKey: "REDIS_PASSWORD"
```

Comment:
Can we set these to an empty dict, and set a comment about the expected structure?